src/codegen/sdk/core/symbol.py

```diff
@@ -290,7 +292,91 @@ def move_to_file(
             AssertionError: If an invalid strategy is provided.
         """
         encountered_symbols = {self}
-        self._move_to_file(file, encountered_symbols, include_dependencies, strategy)
+        self._move_to_file(file, encountered_symbols, include_dependencies, strategy, cleanup_unused_imports)
+
+    @noapidoc
```

Comment:
I like the idea of splitting out move symbol to file! However, it's a pretty complicated function, could we split the refactor from the new feature into separate prs?